feat(highlight): update default color scheme (#26334)
neovim/neovim: runtime/doc/vim_diff.txt

  ":filetype off" to |init.vim|.
- Syntax highlighting is enabled by default. This can be disabled by adding
  ":syntax off" to |init.vim|.

> - Default color scheme is different. This can be disabled by adding

Reviewer: I don't think "disabled" is the right word here.

> - Default color scheme is different. This can be disabled by adding

Reviewer 2: ```suggestion
- Default color scheme has been updated. This can be reversed by adding
```
